### ISSUE TITLE
windows 11, after a while of use or when pressing several times the start menu nvda no longer announces the focus on start 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#12809]
### Steps to reproduce:
I noticed this problem during regular use of windows 11, however I managed to reproduce the problem by opening the start menu several times.
Open the start menu nvda announces the start menu as expected.
Now press esc and nvda announces focus off start as expected.
Now open it and open the start menu again. 
### Actual behavior:
nvda is muted, does not announce focus in start menu.
The menu was opened correctly because doing a search even with nvda muted opens the search result. 
### Expected behavior:
nvda should announce every time the start menu gains focus.
If you cannot reproduce the problem with the steps like this, repeat at least a few more times, with me I can produce the problem even with great frequency. 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-23668,d624b5a6

#### Windows version:
windows 11 10.0.22000.176
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log:
[start.txt](https://github.com/nvaccess/nvda/files/7108692/start.txt)

**Comment by leonardder** [comment]
I can reproduce this as well.

**Comment by leonardder** [comment]
It looks like in these cases, event_UIA_window_windowOpen is fired but the search window doesn't get focus.

**Comment by josephsl** [comment]
Hi,

Apparently resolved, according to release notes for build 22000.282 (beta/release preview at the moment):
https://blogs.windows.com/windows-insider/2021/10/15/releasing-windows-11-build-22000-282-to-beta-and-release-preview-channels/

Thanks.

**Comment by mazen428** [comment]
Hello.
I still get this issue sometimes, it seems like NVDA just does not recognize the search box, I can press tab to navigate the start menu, but when focus is on the search box NVDA just says nothing.
I'm on 22000.318.

**Comment by fernando-jose-silva** [comment]
hello, i still have this problem a lot, when i open the start menu, simply annie ver dear is muted, or at all is open, and if i type any search it is done normally but nvda is muted, windows updates have improved this symptom a lot, but I still have this problem quite often.  I would very much like the nvda team to continue investigating this issue.

Enviado do meu iPhone

**Comment by fernando-jose-silva** [comment]
I would like to share my findings
If I restart nvda it will stop reading the search field in the start menu.
If I restart explorer nvda returns to reading the start menu.
This doesn't occur in jaws, I can turn jaws off and back on and it continues to read the search field in the start menu.
in short, only nvda requires that explorer be restarted after restarting the player.
Can someone please pay attention to this problem, this is very annoying. 
log
[startmenu.txt](https://github.com/nvaccess/nvda/files/7591178/startmenu.txt)

**Comment by CyrilleB79** [comment]
@seanbudd would you mind reopening this ticket.
There is still activity in it but, this ticket being closed, maybe it is not correctly tracked.

**Comment by mazen428** [comment]
I've noticed if you were encounter that bug, you can turn on narrator and NVDA will focus correctly on the start menu search box, but when you turn off narrator, the old behavior will return. Quite strange.

**Comment by fernando-jose-silva** [comment]
Sorry for the confusing report, but still trying to find a solution to the problem I started to modify several options in my nvda settings.
I started by deleting the nvda settings and configuring configuration by configuration, and testing if there was any difference.
The conclusion was very confusing changing some settings had an effect, however in my mind it doesn't make sense that they have any effect on this issue.
Other settings changed may also have an effect when changed, but after a while the problem returns, which seems to be some temporary error that comes and goes when something happens, but I have no idea of ​​the trigger.
The most noticeable trigger is when I restart nvda, when I restart nvda it is almost certain that the problem manifests itself.
Restart the explorer to resolve for a while.
I created a .bat file, restarts nvda and explorer, and even associated a shortcut key, when the problem starts I use the shortcut, nvda and explorer restart and work again for a while until the problem returns.
I tested it when nvda does not read the start search field, it seems that the narrator also stops reading, which contrasts with a report by a colleague on this problem who reported that when he enables the narrator the problem does not manifest itself on his machine, with me if at the moment nvda doesn't read the start search field i call jaws it is normal.
Let's go to the settings that I changed and try to reduce the occurrence of the problem, remembering that it didn't eliminate the problem, it still continues.
I left the option to open the welcome window enabled when starting nvda, normally I disable it.
Allow the NVDA Project to collect NVDA usage statistics
is unchecked, usually marked, but when I unchecked it seemed that the problem manifested itself less frequently.
Simple exploration mode is checked, usually deicho checked.
Thanks for the patience.

**Comment by mazen428** [comment]
Hello. Have you try updating to build 22000.348? I no longer encounter the issue anymore, or if I do, just reopening the start menu fixes it. Thanks.

**Comment by fernando-jose-silva** [comment]
yes, i have updated windows, and the problem still occurs, at a lower frequency than before the update, but it still occurs.
 In my case, to solve the problem momentarily, I don't just close the start menu, and reopen it, the need to restart the screen reader nvda, and sometimes I also have to restart explorer.exe, thank you very much.

**Comment by josephsl** [comment]
Hi,

Reminder: please test with build 22000.652, and for some, build 22610 (22H2 preview) with latest NVDA alpha/beta/release versions and see if the issue is resolved.

Thanks.

**Comment by fernando-jose-silva** [comment]
I am not in a position to test this, out of displeasure I became a User of software such as explorerpatcher and open shell, if the problem depends only on my tests we can close

**Comment by Adriani90** [comment]
@LeonarddeR are you seeing this issue still with NvDA 2024.1 Beta 9 or last alpha?
I didn't see this for a very long while now.

**Comment by LeonarddeR** [comment]
nope, it seems it fixed itself somehow.

**Comment by XLTechie** [comment]
I have seen this as recently as three days ago, on Win 10 (22H2), NVDA 2023.3 
(whatever latest point release of 2023.3 is).

A customer machine I do not have current access to in order to debug further, 
but this can still be observed in the wild.
